hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,208 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_otp_ctrl
+  testpoints: [
+    // OTP (pre-verified IP) integration tests:
+    {
+      name: chip_otp_ctrl_init
+      desc: '''Verify the OTP ctrl initialization on chip power up.
+
+            Verify that the chip powers up correctly on POR.
+            - The pwrmgr initiates a handshake with OTP ctrl and later, with LC ctrl in subsequent
+              FSM states. Ensure that the whole power up sequence does not hang.
+            - Verify with connectivity assertion checks, the handshake signals are connected.
+            - Ensure that no interrupts or alerts are triggered.
+            '''
+      features: ["OTP_CTRL.INIT"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_lc_ctrl_transition"]
+    }
+    {
+      name: chip_sw_otp_ctrl_keys
+      desc: '''Verify the proliferation of keys to security peripherals.
+
+            - Verify the correctness of keys provided to SRAM ctrl (main & ret), flash ctrl, keymgr,
+              (note that keymgr does not have handshake).
+            - Ensure that the test requests a new key and verifies the previously written
+              data to an address now returns a garbage value.
+
+            X-ref'ed with the following IP tests that consume these signals:
+            - chip_sw_sram_scrambled_access
+            - chip_sw_flash_scramble
+            - chip_sw_keymgr_key_derivation
+            - chip_sw_otbn_mem_scramble
+            '''
+      features: ["OTP_CTRL.KEY_DERIVATION", "OTP_CTRL.PARTITION.SECRET1"]
+      stage: V2
+      si_stage: SV1
+      tests: [// Verifies both, main and retention SRAM scrambling.
+              "chip_sw_sram_ctrl_scrambled_access",
+              "chip_sw_flash_init",
+              "chip_sw_keymgr_key_derivation",
+              "chip_sw_otbn_mem_scramble",
+              "chip_sw_rv_core_ibex_icache_invalidate"]
+    }
+    {
+      name: chip_sw_otp_ctrl_entropy
+      desc: '''Verify the entropy interface from OTP ctrl to EDN.
+
+            This is X-ref'ed with the chip_otp_ctrl_keys test, which needs to handshake with the EDN
```

Comment:
Same as the test above.

Reply:
https://github.com/lowRISC/opentitan/pull/19889#discussion_r1373814018

Reply:
Please state the check in terms of sival observable side-effects, since we won't be able to look at internal signals, as the sim_dv test probably does.